src/org/folio/Constants.groovy

```diff
@@ -44,8 +44,8 @@ class Constants {
     static String FOLIO_HELM_HOSTED_REPO_URL = "https://repository.folio.org/repository/helm-hosted/"
     static String FOLIO_NPM_REPO_URL = "https://repository.folio.org/repository/npm-folioci/"
 
-    static String FOLIO_HELM_V2_REPO_URL= "https://repository.folio.org/repository/folio-helm-v2/"
-    static String FOLIO_HELM_V2_REPO_NAME = "folio-helm-v2"
+    static String FOLIO_HELM_V2_REPO_URL= "https://repository.folio.org/repository/folio-helm-v2-test/"
```

Comment:
What was the purpose of this change? If it was just for testing don't forget to change it before merging.

Reply:
For testing
Resolved